requirements.txt

```diff
@@ -10,5 +10,6 @@ torch==1.2.0
 torchvision==0.2.1
 tqdm>=4.30
 future
+pillow==6.1.0
```

Comment:
is that related to this PR?

Reply:
It fixed a bug with a new version of **Pillow** (which leads tests to fail). I can move it to another PR if you prefer.

Reply:
> I can move it to another PR if you prefer.

it's fine, but just mention it in the body of the PR so reviewers understand this is not related to the main purpose of this PR